inference/rv_stac_config.py

```diff
@@ -93,62 +97,60 @@ def build_dataset_from_catalog(
 ) -> DatasetConfig:
 
     # Read taining scenes from STAC
-    train_collection = catalog.get_child(id="train")
+    train_collection = catalog.get_child(id="training_imagery")  # ???
+    train_collection = catalog.get_child(id="training_imagery")
     train_scenes = make_scenes(train_collection, channel_order)
 
-    # Read testing scenes from STAC
-    test_collection = catalog.get_child(id="test")
-    test_scenes = make_scenes(test_collection, channel_order)
-
     # Read validation scenes from STAC
-    validation_collection = catalog.get_child(id="validation")
+    validation_collection = catalog.get_child(id="validation_imagery")
     validation_scenes = make_scenes(validation_collection, channel_order)
 
+    # Read testing scenes from STAC
+    test_collection_0 = catalog.get_child(id="test_imagery_0")
+    test_scenes_0 = make_scenes(test_collection_0, channel_order)
+    test_collection_1 = catalog.get_child(id="test_imagery_1")
+    test_scenes_1 = make_scenes(test_collection_1, channel_order)
+
     return DatasetConfig(
         class_config=class_config,
         train_scenes=train_scenes,
-        test_scenes=test_scenes,
+        test_scenes=test_scenes_0,
         validation_scenes=validation_scenes,
     )
 
 
-def get_config(runner):
-    output_root_uri = os.environ.get("OUTPUT_ROOT_URI")
-    if output_root_uri is None:
-        sys.exit("No output URI set. Set the environment variable OUTPUT_ROOT_URI")
-
-    catalog_root = os.environ.get("CATALOG_ROOT_URI")
-    if catalog_root is None:
-        sys.exit("No catalog URI set. Set the environment variable CATALOG_ROOT_URI")
-
+def get_config(runner, root_uri, catalog_root, hours):
     # Read STAC catalog
     catalog: Catalog = Catalog.from_file(catalog_root)
 
     # TODO: pull desired channels from root collection properties
-    channel_ordering: [int] = [0, 1, 2]
+    channel_ordering: [int] = [0, 1, 1]
```

Comment:
Is this duplication intentional?

Reply:
Yeah, I discovered that the multiband functionality does kick-in until the number of channels is > 3.  The totally pretrained models expect 3 channels, so one is forced to duplicate.